release-notes/README.md

```diff
@@ -6,6 +6,10 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 ## June 7, 2022
 
+### iOS & Android Identity 1.1.0
```

Comment:
Let's break this into two different labels to easily search through release notes.

iOS AEPEdgeIdentity 1.1.0
Android EdgeIdentity 1.1.0